pkg/cronjob/es_rollover.go

```diff
@@ -201,6 +201,7 @@ func EsScriptEnvVars(opts v1.Options) []corev1.EnvVar {
 		{flag: "es.username", envVar: "ES_USERNAME"},
 		{flag: "es.password", envVar: "ES_PASSWORD"},
 		{flag: "es.tls", envVar: "ES_TLS"},
```

Comment:
can we remove this line? @indigostar-kr

Reply:
Fix #1901

Reply:
If you want, I'll merge the #1900 and #1901.

Reply:
lets close #1900

Reply:
ok